src/main/java/com/google/sps/image/ImageSelection.java

```diff
@@ -45,7 +65,10 @@ public String generateSearchUrl() {
         break;
       }
     }
-    bingUrl = bingUrl + BING_QUERY_PARAM;
+    
+    bingUrl = addBingQueryParam(bingUrl);
+    bingUrl = addLicenseFilter(bingUrl);
+    bingUrl = addSafeSearchFilter(bingUrl);
```

Comment:
It's not really clear to me why you are calling three times technically something that can be done in 1 function.
If the purpose is that in the future you can call them independently, this could be anyway just 1 function since basically the 3 functions do the same thing.
addAdditionalFilter(String url, String filter)
having the filter passed when called.

Reply:
Done